CHANGELOG.md

```diff
@@ -24,6 +24,7 @@ Features:
 - [#1340](https://github.com/rails-api/active_model_serializers/pull/1340) Add support for resource-level meta. (@beauby)
 
 Fixes:
+- [#1622](https://github.com/rails-api/active_model_serializers/pull/1622) Fix inherited serializers when using fragment cache
```

Comment:
`Fix fragment caching inheriting serializers to use distinct per-serializer caches. (@Iserman)`